app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -300,7 +294,8 @@ sealed class Event {
         enum class Item {
             SETTINGS, LIBRARY, HELP, DESKTOP_VIEW_ON, DESKTOP_VIEW_OFF, FIND_IN_PAGE, NEW_TAB,
             NEW_PRIVATE_TAB, SHARE, REPORT_SITE_ISSUE, BACK, FORWARD, RELOAD, STOP, OPEN_IN_FENIX,
-            SAVE_TO_COLLECTION, ADD_TO_HOMESCREEN, QUIT
+            SAVE_TO_COLLECTION, ADD_TO_HOMESCREEN, QUIT, READER_MODE_ON, READER_MODE_OFF, OPEN_IN_APP,
```

Comment:
I'm _not sure_ but we **may** need a data review for these 🤔  @boek or @liuche to confirm.

Reply:
Good point, I'll add a review request.